EIPS/eip-3756.md

```diff
@@ -0,0 +1,55 @@
+---
+eip: 3756
+title: Gas Limit Cap
+description: Set an in-protocol cap for the gas limit
+author: lightclient (@lightclient)
+discussions-to: https://ethereum-magicians.org/t/eip-3756-gas-limit-cap/6921
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-08-21
+---
+
+## Abstract
+
+Set an in-protocol cap for the gas limit of 30,000,000.
+
+## Motivation
+
+A high gas limit increases pressure on the network. In the benign case, it
+increases the size of the state and history faster than we can sustain. In the
+malicious case, it amplifies the devestation of certain denial-of-service
+attacks.
+
+## Specification
+
+As of the fork block `N`, consider blocks with a `gas_limit` greater than
+`30,000,000` invalid.
```

Comment:
```suggestion
As of FORK_BLOCK_NUMBER, consider blocks with a `gas_limit` greater than `30,000,000` invalid.
```
This is the style we usually use for referring to the fork block number inline.  Unless there is an argument for deviating from that I would prefer to stick to the normal style here (note: I also personally like the previous style, so I'm a bit biased).